client/reader/stream/_style.scss

```diff
@@ -595,14 +595,18 @@ $reader-related-card-v2-breakpoint-small: "( max-width: 535px )";
 		}
 
 		.reader-related-card-v2__post {
-			max-height: 220px;
+			max-height: 200px;
 
 			@media #{$reader-related-card-v2-breakpoint-small} {
-				max-height: 170px;
+				max-height: 150px;
 			}
 
 			@media #{$reader-related-card-v2-breakpoint-medium} {
-				max-height: 170px;
+				max-height: 150px;
+			}
+
+			.reader-related-card-v2__excerpt {
+				-webkit-line-clamp: 3;
```

Comment:
Can we utilize a different property than `-webkit-line-clamp` because it isn't cross-browser compatible?

Maybe a `max-height` and/or `long-content-fade`?